extension/jaegerremotesampling/extension_test.go

```diff
@@ -70,6 +70,7 @@ func TestStartAndShutdownRemote(t *testing.T) {
 
 	// create the config, pointing to the mock server
 	cfg := testConfig()
+	cfg.GRPCServerSettings.NetAddr.Endpoint = "127.0.0.1:0"
```

Comment:
hm. Isnt that default?

Reply:
no, the value set in testConfig() the line above does this:
```
	cfg := createDefaultConfig().(*Config)
	cfg.HTTPServerSettings.Endpoint = "127.0.0.1:5778"
	cfg.GRPCServerSettings.NetAddr.Endpoint = "127.0.0.1:14250"
	return cfg
```
The errors we get are:
`error while starting the gRPC server: failed to listen on gRPC port: listen tcp :14250: bind: address already in use`